Test case failed

```bash
websocket/Tests/WebSocketTests/WebSocketTests.swift:106: error: 
-[WebSocketTests.WebSocketTests testServerContinuation] : 
XCTAssertEqual failed: ("/") is not equal to ("!dlrow ,olleH")
```

--- a/Tests/WebSocketTests/WebSocketTests.swift
+++ b/Tests/WebSocketTests/WebSocketTests.swift
@@ -84,7 +84,6 @@ class WebSocketTests: XCTestCase {
         let ws = HTTPServer.webSocketUpgrader(shouldUpgrade: { req in
             return [:]
         }, onUpgrade: { ws, req in
-            ws.send(req.url.path)
             ws.onText { ws, string in
                 ws.send(string.reversed())
             }

--- a/circle.yml
+++ b/circle.yml
@@ -7,6 +7,9 @@ jobs:
     steps:
       - checkout
       - run:
+          name: Install libressl
+          command: brew install libressl
+      - run:
           name: Compile code
           command: swift build
       - run:

--- a/circle.yml
+++ b/circle.yml
@@ -3,11 +3,18 @@ version: 2
 jobs:
   macos:
     macos:
-      xcode: "10.0.0"
+      xcode: "10.2"
     steps:
       - checkout
-      - run: swift build
-      - run: swift test
+      - run:
+          name: Compile code
+          command: swift build
+      - run:
+          name: Run unit tests
+          command: swift test
+      - run:
+          name: Compile code with optimizations
+          command: swift build -c release
 
   linux:
     docker:
@@ -24,32 +31,12 @@ jobs:
           name: Compile code with optimizations
           command: swift build -c release
 
-
-  linux-vapor:
-    docker:
-      - image: codevapor/swift:4.1
-    steps:
-      - run:
-          name: Clone Vapor
-          command: git clone -b master https://github.com/vapor/vapor.git
-          working_directory: ~/
-      - run:
-          name: Switch Vapor to this WebSocket revision
-          command: swift package edit WebSocket --revision $CIRCLE_SHA1
-          working_directory: ~/vapor
-      - run:
-          name: Run Vapor unit tests
-          command: swift test
-          working_directory: ~/vapor
-
-
 workflows:
   version: 2
   tests:
     jobs:
       - linux
-      - linux-vapor
-      # - macos
+      - macos
 
   nightly:
     triggers:
@@ -61,4 +48,4 @@ workflows:
                 - master
     jobs:
       - linux
-      # - macos
+      - macos

